### DISCUSSION TITLE
List organization repositories to which you have admin access

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,

Is there a way for a user to view all organization repositories to which he has admin/owner access?

I guess he can use this list https://github.com/settings/repositories but here I think you see all repositories where you are collaborator (whether you have only write, read or maintain access) and the list can be quite long in an organization. So that's why I need a way a user to filter somehow easily only those to which he has admin access.

Thank you for your help.

**Comment by ossanna16** [comment]
Hi there @marielaSAG and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by andy-maier** [comment]
@marielaSAG  @ossanna16  I ran into the same question. In case anyone has an answer, it would be good to post here.

**Comment by drewcdisc** [comment]
Bump. Keep the question alive.

I'm looking for the same thing today.

**Comment by mariestaver** [comment]
Looking for this today, myself. I'm also looking for a list of ALL admins on all the repositories in my organization, but seeing my own admin access would be a start.
